projects/angular/src/utils/key-codes/key-codes.ts

```diff
@@ -8,9 +8,4 @@ export const UP_ARROW = 38;
 export const DOWN_ARROW = 40;
 export const RIGHT_ARROW = 39;
 export const LEFT_ARROW = 37;
-export const ENTER = 13;
-export const SPACE = 32;
-export const SHIFT = 16;
-export const TAB = 9;
-export const ESC = 27;
```

Comment:
Same here:
As this is a utility, I'd prefer to keep the unused enumeration values for completeness.
It's not a big overhead.

Reply:
These aren't used at all anymore since I changed everything to `event.key` instead of `event.keyCode`.